src/store/modules/events/index.ts

```diff
@@ -538,21 +580,72 @@ const module: Module<EventsModuleState, RootState> = {
         return;
       }
 
-      const repetition = event.repetition;
+      let repetition;
```

Comment:
```suggestion
      let repetition = event.repetition;
```
then you can remove the else condition below

Reply:
done